tests/test_bitmap_font.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2014-18 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+"""
+Tests for the :py:module:`luma.core.bitmap_font` module.
+"""
+
+import os
+import cbor2
+import pytest
+from PIL import Image, ImageDraw
+from luma.core import bitmap_font
+from luma.core.bitmap_font import embedded_fonts
+
+from helpers import get_reference_pil_font, get_reference_file
+
+
+FONTDATA = {
+    # TODO: Extend and verify Unicode Mappings
```

Comment:
if this todo is not happening/resolved in this PR, please open a ticket and reference that ticket nr here.

Reply:
Have removed reference to a TODO.  It's actually for the data that will be in luma.lcd.const and luma.oled.const.  I will make sure to open issues for both when I issue the PR for them.